AnKing-VIP/AnKing-Note-Types
# [BUILD-873] feat: Add MH link buttons

On mobile, we need to show the new “Boards and Beyond Links” and “First Aid Links” buttons on cards (which have BB/FA tags).

We will need some javascript on the note type that converts the BB/FA tags into urls and adds them to the card (under new buttons).

## Related issues
https://ankihub.atlassian.net/jira/software/c/projects/BUILD/boards/1?selectedIssue=BUILD-876

## How to reproduce
- Add these tags to a note:
  ```
  #AK_Step1_v12::#B&B::01_Basic_Pharmacology::01_General::01_Enzymes
  #AK_Step1_v12::#B&B::01_Basic_Pharmacology::01_General::02_Enzyme_Inhibitors
  #AK_Step1_v12::#B&B::01_Basic_Pharmacology::01_General::03_Dose-Response
  #AK_Step1_v12::#FirstAid::01_Biochem::06_Metabolism::31_Alkaptonuria
  #AK_Step1_v12::#FirstAid::01_Biochem::06_Metabolism::32_Homocystinuria
  #AK_Step1_v12::#FirstAid::01_Biochem::06_Metabolism::33_Cystinuria
  ```
- Open the note in the reviewer.
- Click on Edit -> Cards to open the Card template editor
- Paste `AnKingOverhaul/Back Template.html` into the Back Template editor and `AnKingOverhaul/Styling.css` into the Styling editor
- Remove this (just added) css rule from the Styling editor:
  ```
  /* Show tag based hint buttons only on mobile */
  html:not(.mobile) .tagBasedHintBtn {
    display: none;
  }
  ```
  - This way you will be able to see the new buttons on Desktop
- When you click on the buttons, the B&B and FA4 links should show up, as shown on the screenshot below
- The links should use correct urls


## Screenshots
<img src="https://github.com/user-attachments/assets/ce8efbf0-6372-4b87-bc4a-8ba83b507f32" width="500" />

## Comments
I checked if the the note type works with the note types add-on. The new fields are not configurable, but it doesn't break anything.
